src/drivers/github.js

```diff
@@ -328,9 +335,81 @@ class Github {
       body
     });
 
+    if (autoMerge) {
+      await this.prAutoMerge({ pullRequestId: nodeId, base });
+    }
+
     return htmlUrl;
   }
 
+  /**
+   * @param {string} branch
+   * @returns {Promise<boolean>}
+   */
+  async isProtected(branch) {
```

Comment:
should receive an opts object

Reply:
I'd prefer destructuring for consistency.

```suggestion
  async isProtected({ branch }) {
```

Reply:
addressed in https://github.com/iterative/cml/pull/903/commits/a44d3c1850eba4192763fa310e39a7bfb60a6ba2

Reply:
Thank you!